dask/utils.py

```diff
@@ -1369,3 +1369,43 @@ def ndimlist(seq):
         return 1
     else:
         return 1 + ndimlist(seq[0])
+
+
+from .delayed import delayed
+from operator import add
+
+
+def sequence(a, b, combine=None):
```

Comment:
I think we should avoid adding these to the public API, at least without more discussion. If you need these for to_sql you can make them private for now.